src/js/tech/html5.js

```diff
@@ -282,6 +293,33 @@ class Html5 extends Tech {
     this.textTracks().removeTrack_(e.track);
   }
 
+  removeOldTextTracks() {
+    // This will loop over the texttracks and check if they are still used
+    // If not, they will be removed from the emulated list
+    let removeTracks = [];
+
+    for (let i = 0; i < this.textTracks().length; i++) {
```

Comment:
I kinda do like the other short version, but this one isn't really readable anymore :yum: